utils/common/src/test/java/org/apache/brooklyn/util/javalang/ReflectionsTest.java

```diff
@@ -24,12 +24,12 @@
 
 import java.lang.reflect.Field;
 import java.lang.reflect.Method;
-import java.util.Arrays;
-import java.util.List;
+import java.util.*;
```

Comment:
Convention in brooklyn is to not use `.*` imports.

(If it were for everything in a nested class, such as you do in `org.apache.brooklyn.core.catalog.internal.CatalogUpgradeScannerTest.Givens.*;` then that makes complete sense. But for things like `java.util...` I'd prefer explicit imports.